src/main/java/competition/IdealElectricalContract.java

```diff
@@ -21,55 +21,50 @@ public DeviceInfo rightRearDriveNeo() {
     }
 
 
-    public DeviceInfo hoodMotor(){
+    public DeviceInfo hoodMotor() {
         return new DeviceInfo(90, false); //Motor not real
     }
 
-    public boolean isHoodReady(){
+    public boolean isHoodReady() {
         return true;
     }
     
     public boolean isKickerReady(){
         return true;
     }
 
-    public boolean isTurretReady(){
+    public boolean isTurretReady() {
         return true;
     }
 
-    public boolean isShooterWheelReady(){
+    public boolean isShooterWheelReady() {
         return true;
     }
 
 
     //Everything above sure about inversion
     //Everything below not sure about the inversion
-    public DeviceInfo intakeMotor()
-    {
-        return new DeviceInfo(24, false);
+    public DeviceInfo intakeMotor() {
+        return new DeviceInfo(29, false);
```

Comment:
do we need to change this channel?

Reply:
the actual motor ID for intakeMotor is 29, according to the electrical programming contract document

Reply:
Thanks for confirming